pkg/apis/hiveinternal/v1alpha1/clustersync_types.go

```diff
@@ -36,6 +36,10 @@ type ClusterSyncStatus struct {
 	// Conditions is a list of conditions associated with syncing to the cluster.
 	// +optional
 	Conditions []ClusterSyncCondition `json:"conditions,omitempty"`
+
+	// FirstSyncSetsSuccessTime is the time we first successfully applied all (selector)syncsets to a cluster.
+	// +optional
+	FirstSyncSetsSuccessTime *metav1.Time `json:"firstSyncSetsSuccessTime,omitempty"`
```

Comment:
Suggest: FirstSuccessTime and firstSuccessTime. This is on a ClusterSync and thus we're simply saying it's the first time a full ClusterSync was applied.